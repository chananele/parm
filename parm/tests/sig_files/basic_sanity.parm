---
name: basic_sanity
exports:
  - test
method: find_single
pattern: |
  test: bne #0x1043c
...
