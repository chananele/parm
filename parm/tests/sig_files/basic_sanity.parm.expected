name: basic_sanity
result: pass
matches:
  test: 66648
